addons/sale/report/sale_report.py

```diff
@@ -97,6 +97,7 @@ def _select_sale(self):
         select_ = f"""
             MIN(l.id) AS id,
             l.product_id AS product_id,
+            l.price_unit AS price_unit,
```

Comment:
must be converted to the right currency, and adapted to the uom factor too, like the other amounts

Reply:
Done the changes :+1: